<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BibFixer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resource.qrc">
    <normaloff>:/Images/Fix.png</normaloff>:/Images/Fix.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="BibFixer::TextEdit" name="teOutput"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuOperation">
    <property name="title">
     <string>Operation</string>
    </property>
    <addaction name="actionRunAll"/>
    <addaction name="actionClean"/>
    <addaction name="actionCapitalize"/>
    <addaction name="actionProtect"/>
    <addaction name="actionAbbreviate"/>
    <addaction name="actionShortenNames"/>
    <addaction name="actionGenerateKeys"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOperation"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionImport"/>
   <addaction name="actionSave"/>
   <addaction name="actionRunAll"/>
   <addaction name="actionClean"/>
   <addaction name="actionCapitalize"/>
   <addaction name="actionProtect"/>
   <addaction name="actionAbbreviate"/>
   <addaction name="actionShortenNames"/>
   <addaction name="actionGenerateKeys"/>
  </widget>
  <action name="actionCapitalize">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Images/Uppercase.png</normaloff>:/Images/Uppercase.png</iconset>
   </property>
   <property name="text">
    <string>Capitalize</string>
   </property>
   <property name="toolTip">
    <string>Capitalize</string>
   </property>
  </action>
  <action name="actionAbbreviate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Images/Abbreviation.png</normaloff>:/Images/Abbreviation.png</iconset>
   </property>
   <property name="text">
    <string>Abbreviate</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Images/Settings.png</normaloff>:/Images/Settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Images/Save.png</normaloff>:/Images/Save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionClean">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Images/Clean.png</normaloff>:/Images/Clean.png</iconset>
   </property>
   <property name="text">
    <string>Clean</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Images/About.png</normaloff>:/Images/About.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionProtect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Images/Protect.png</normaloff>:/Images/Protect.png</iconset>
   </property>
   <property name="text">
    <string>Protect</string>
   </property>
  </action>
  <action name="actionRunAll">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Images/Run.png</normaloff>:/Images/Run.png</iconset>
   </property>
   <property name="text">
    <string>Run All</string>
   </property>
  </action>
  <action name="actionGenerateKeys">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Images/Key.png</normaloff>:/Images/Key.png</iconset>
   </property>
   <property name="text">
    <string>Generate keys</string>
   </property>
   <property name="toolTip">
    <string>Generate keys</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Images/New.png</normaloff>:/Images/New.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionShortenNames">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Images/Heads.png</normaloff>:/Images/Heads.png</iconset>
   </property>
   <property name="text">
    <string>Shorten Names</string>
   </property>
   <property name="toolTip">
    <string>Shorten Names</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Images/Bibtex.png</normaloff>:/Images/Bibtex.png</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
   <property name="toolTip">
    <string>Import from a Latex file</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BibFixer::TextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>TextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
